tests/forms_tests/field_tests/test_jsonfield.py

```diff
@@ -114,3 +115,10 @@ class JSONForm(Form):
         form = JSONForm({'name': 'xy', 'json_field': '{"foo"}'})
         self.assertEqual(form.errors['json_field'], ['Enter a valid JSON.'])
         self.assertIn('{&quot;foo&quot;}</textarea>', form.as_p())
+
+    def test_bound_data_none(self):
+        class JSONForm(Form):
+            json_field = JSONField(required=True)
+        form = JSONForm({'name': 'xyz'})
```

Comment:
I would add two sub-tests, one with `{}` and second with `{'json_field': None}`.